lib/manager/reducers/status.js

```diff
@@ -71,7 +72,7 @@ const config = (state: StatusState = defaultState, action: Action): StatusState
     case 'UPDATING_SERVER':
     case 'FETCHING_ALL_JOBS':
     case 'REQUEST_RTD_ALERTS':
-      return {...state, message: messages(action.type)}
+      return {...state, message: messages(action.type), isSaving: action.type === 'SAVING_FEEDSOURCE'}
```

Comment:
Is that `isSaving` status getting unset after the the feed source is saved, or through another action above (and which one(s))?

Reply:
The parameter will get set to false after the feed source is saved by any subsequent action in the case list. 
![image](https://user-images.githubusercontent.com/63798641/216098835-a94d01e8-dafc-4dd2-9cf9-8fa621889de5.png)

Reply:
What's setting it back to false? Do you think this could be the source of the ci failures

Reply:
From looking at the code, add to the comment on line 75 that `REQUESTING_FEED_SOURCE` and `REQUESTING_PROJECT` set the `isSaving` state back to false, which is what we want.